dsnp-wallet/dsnp-wallet/Source/ViewControllers/SettingsViewController.swift

```diff
@@ -7,25 +7,35 @@
 
 import Foundation
 import UIKit
-import DSNPWallet
 
 class SettingsViewController: UIViewController {
     override func viewDidLoad() {
         setViews()
     }
     
     @objc func tappedLogOut(selector: UIButton?) {
-        do {
-            let _ = try DSNPWallet().deleteKeys()
-        } catch {
-            let alert = UIAlertController(title: "Error deleting keys", message: nil, preferredStyle: .alert)
-            alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
-            present(alert, animated: true, completion: nil)
+        let alert = UIAlertController(title: "Logging out will clear your keys.", message: nil, preferredStyle: .alert)
+        
+        let okAction = UIAlertAction(title: "OK", style: .default, handler: { _ in
+            do {
+                let _ = try AuthManager.shared.logout()
+            } catch {
+                let alert = UIAlertController(title: "Problem deleting keys, log out failed.", message: nil, preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+                self.present(alert, animated: true, completion: nil)
+            }
+            
+            self.presentGenKeysVC()
```

Comment:
Does it make sense to move this in the do { } after logout? Currently, if there is an error logging out, we present the error alert, then we attempt presenting the genKeysVC